structure/templates/structure_browser.html

```diff
@@ -302,7 +290,13 @@ <h2 style="width:auto; display:inline;">Structures</h2>
                             No
                             {% endif %}
                         </td>
-                        <td>{{ structure.publication.authors|last_author}}</td>
+                        {% if structure.publication.authors %}

```

Comment:
Nothing wrong, but a suggestion for consistency:
Do the if-else within the TD instead of duplicating these tags.